.github/workflows/update-snippets.yml

```
@@ -0,0 +1,70 @@
+name: Update Snippets
+# Update cylc-snippets.json using bin/cylc-snippets.
+
+on:
+  # Ideally people making changes should re-run this task, but probably
+  # won't remember - I think once a month is probably often enough.
+  schedule:
+    - cron: "* * 15 * *"
```

https://crontab.guru/#*_*_15_*_*

> “At every minute on day-of-month 15.”

```suggestion
    - cron: "51 03 15 * *" # 03:51 on the 15th of the month
```